src/kiutils/items/schitems.py

```diff
@@ -816,9 +820,23 @@ def from_sexpr(cls, exp: list) -> SchematicSymbol:
             if item[0] == 'at': object.position = Position().from_sexpr(item)
             if item[0] == 'property': object.properties.append(Property().from_sexpr(item))
             if item[0] == 'pin': object.pins.update({item[1]: item[2][1]})
+            if item[0] == 'instances': object.instances = cls.parse_project_instances(item)
```

Comment:
Here we could loop over all tokens in `item[1]` to get to all project instances:
```python
if item[0] == 'instances':
  for instance in item[1]:
    object.instances.append(ProjectInstanceNameToBeDefined.from_sexpr(instance))
```